tests/internal/params/vars.go

```diff
@@ -1,9 +1,11 @@
 package params
 
 // PrivilegedNSLabels represents privileged labels.
-var PrivilegedNSLabels = map[string]string{
-	"pod-security.kubernetes.io/audit":               "privileged",
-	"pod-security.kubernetes.io/enforce":             "privileged",
-	"pod-security.kubernetes.io/warn":                "privileged",
-	"security.openshift.io/scc.podSecurityLabelSync": "false",
-}
+var (
```

Comment:
Why did you change it? Can you revert it back to it's original state?

Reply:
Fixed. I added a variable here before and reformatted.